### PR TITLE
Style Changes

### PR DESCRIPTION
**Pull request by jwang1139**
Added comments and moved brackets to the same line of constructor functions in encoder.cpp, RadioReceiver.cpp, and rbserialmessages.cpp

### CODE REVIEW

File: real_time/arduino_src/lib_avr/rbserialmessages/rbserialmessages.cpp [hunk 1]
```diff
@@ -18,13 +18,10 @@
 
 // Public //////////////////////////////////////////////////////////////////////
 
-RBSerialMessages::RBSerialMessages() 
-{
-}
+RBSerialMessages::RBSerialMessages() {} //constructor function
```
Comment (ramensandwich) [inline real_time/arduino_src/lib_avr/rbserialmessages/rbserialmessages.cpp]: Based on how cpp is, I don't think it's necessary to clarify that these empty functions are constructors

File: real_time/arduino_src/radio_buggy_mega/main.cpp [hunk 2]
```diff
@@ -338,46 +337,39 @@ void indicator_light_init()
 }
```
Comment (ramensandwich) [inline real_time/arduino_src/radio_buggy_mega/main.cpp]: Can we put an @brief at the beginning of each of these lighting functions like we did for the indicator light?

File: real_time/arduino_src/radio_buggy_mega/Makefile [hunk 3]
```diff
@@ -533,7 +533,7 @@ clean_list :
 	$(REMOVE) $(patsubst %.c,%.d,$(patsubst %.cpp,%.d,$(SRC)))
 	$(REMOVE) .dep/*
 	$(REMOVE) fingerprint.h
-	$(REMOVE) rbsm_config.h
+	$(REMOVE) ../lib_avr/rbserialmessages/rbsm_config.h
```
Comment (grypoB) [inline real_time/arduino_src/radio_buggy_mega/Makefile]: not sure this is the right path

File: real_time/arduino_src/radio_buggy_mega/main.cpp [hunk 4]
```diff
@@ -153,21 +154,18 @@ inline long map_signal(long x,
...
-                  long lower)
-{
+                  long lower) {
     return (max(min(input, upper), lower));
 }
 
 
-void adc_init(void) // copy-pasted from wiring.c l.353 (Arduino library)
-{
+void adc_init(void) {// copy-pasted from wiring.c l.353 (Arduino library)
```
Comment (grypoB) [inline real_time/arduino_src/radio_buggy_mega/main.cpp]: add space between { and //

File: real_time/arduino_src/radio_buggy_mega/main.cpp [hunk 5]
```diff
@@ -507,16 +509,14 @@ int main(void)
         int read_status;
 
         while((read_status = g_rbsm.Read(&new_command))
-             != RBSM_ERROR_INSUFFICIENT_DATA) 
-        {
-            if(read_status == 0) 
-            {
+             != RBSM_ERROR_INSUFFICIENT_DATA) {
```
Comment (grypoB) [inline real_time/arduino_src/radio_buggy_mega/main.cpp]: wierd indentation

File: real_time/arduino_src/radio_buggy_mega/main.cpp [hunk 6]
```diff
@@ -634,26 +634,22 @@ int main(void)
...
         // Set outputs
-        if(g_is_autonomous)
-        {
+        if(g_is_autonomous) {
             steering_set(auto_steering_angle);
             g_rbsm.Send(RBSM_MID_MEGA_STEER_ANGLE, (long int)(auto_steering_angle));
         }
-        else
-        {
+        else {
```
Comment (grypoB) [inline real_time/arduino_src/radio_buggy_mega/main.cpp]: } else { on a single line

File: real_time/arduino_src/radio_buggy_mega/main.cpp [hunk 7]
```diff
@@ -634,26 +634,22 @@ int main(void)
...
            (g_is_autonomous == true && brake_cmd_auton_engaged == true) ||
-           brake_needs_reset == true)
-        {
+           brake_needs_reset == true) {
             brake_drop();
             g_rbsm.Send(RBSM_MID_MEGA_BRAKE_STATE,(long unsigned)true);
         } 
-        else 
-        {
+        else {
```
Comment (grypoB) [inline real_time/arduino_src/radio_buggy_mega/main.cpp]: same

File: real_time/arduino_src/radio_buggy_mega/main.cpp [hunk 8]
```diff
@@ -507,16 +509,14 @@ int main(void)
         int read_status;
 
         while((read_status = g_rbsm.Read(&new_command))
-             != RBSM_ERROR_INSUFFICIENT_DATA) 
-        {
-            if(read_status == 0) 
-            {
+             != RBSM_ERROR_INSUFFICIENT_DATA) {
+
```
Comment (grypoB) [inline real_time/arduino_src/radio_buggy_mega/main.cpp]: nervermind

**Pull request merged**